humanguide/other-aspects/05-translations.rst

```diff
@@ -0,0 +1,21 @@
+Translations
+============
+
+As part of the accessibility, translating an app is important.
+`Clickable <http://clickable.bhdouglass.com/>`__ will handle this
+process automatically, extracting new translatable strings into a pot
+file (the source strings of your app, to be translated)
```

Comment:
"Adding translations to your app makes it more accessible to international users. Clickable..."